backend/src/main/java/com/tradomator/service/WalletService.java

```diff
@@ -0,0 +1,40 @@
+package com.tradomator.service;
+
+import com.tradomator.model.Wallet;
+import com.tradomator.repo.WalletRepo;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+
+@Service
+public class WalletService {
+
+    private final WalletRepo walletRepo;
+
+
+    public WalletService(WalletRepo walletRepo) {
```

Comment:
Constructor could be generated.

Reply:
Cool, will directly try: Yes test still positive. Application still runs. Nice one. Thanks. I took the required Args as you suggested previously.